substreams/ethereum-uniswap-v4/with-hooks/src/variant_modules/3_store_pools_per_hook.rs

```diff
@@ -0,0 +1,39 @@
+use substreams::store::{StoreAppend, Appender};
+use substreams_helper::hex::Hexable;
+use tycho_substreams::prelude::*;
+
+#[substreams::handlers::store]
+pub fn store_pools_per_hook(
+    _params: String,
+    pools_created: BlockEntityChanges,
+    output: StoreAppend<String>,
```

Comment:
Almost certain that this store is going to hit limits quickly. Why is it needed?